web/db/migrations/20260730200000_iroh_binding_discovery_pages/migration.sql

```
@@ -0,0 +1,3 @@
+CREATE INDEX "iroh_endpoint_bindings_user_active_page_idx"
+  ON "iroh_endpoint_bindings" ("user_id", "id")
+  WHERE "revoked_at" IS NULL;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)